src/dsp_tools/cli_entry_point.py

```diff
@@ -114,7 +114,7 @@ def _make_parser(
         help="The links in the XML file point to IRIs (on the server) instead of IDs (in the same XML file).",
     )
     parser_upload.add_argument(
-        "-V", "--validate", action="store_true", help="validate the XML file without uploading it"
+        "-V", "--validate-only", action="store_true", help="validate the XML file without uploading it"
```

Comment:
This is a breaking change. I did it for consistency reasons.

Reply:
makes sense. Note, though, that the  PR name does not have the exclamation mark for breaking change, so unless you have a different release-please setup here, this will not appear as a major version bump